Stackdriver-Webhook/README.md

```diff
@@ -0,0 +1,184 @@
+
+
+# Catchpoint Google Stackdriver
+
+  
+
+####
+
+## **Introduction**
+
+  
+
+Cloud Monitoring provides visibility into the performance, uptime, and overall health of applications. It collects metrics, events, and metadata from Google Cloud, Amazon Web Services, hosted uptime probes, application instrumentation, and a variety of common application components including Cassandra, Nginx, Apache Web Server, Elasticsearch, and many others. Operations ingests that data and generates insights via dashboards, charts, and alerts. Cloud Monitoring alerting helps you collaborate by integrating with Slack, PagerDuty, and more.
+
+  
+
+####
+
+  
+  
+
+We will be looking at ingesting data from Catchpoint into Cloud Monitoring.
```

Comment:
Should we mention if we will be using REST API or webhook approach in this integration, also give link to Catchpoint KB like you have given for Google Cloud.

Reply:
Added telling its webhook.
Still waiting for kb to go live for Catchpoint KB